sdk/typescript/src/framework/framework.ts

```diff
@@ -151,7 +151,6 @@ export type DelegationData = SuiMoveObject & {
       id: string;
       version: number;
     };
-    coin_locked_until_epoch: Option<SuiMoveObject>;
```

Comment:
you should keep this and add a `// TODO: clean up after 0.34` for backward compatibility

Reply:
Done.